src/components/export/export.js

```diff
@@ -1,46 +1,49 @@
-import { useWalletSelector } from '../../contexts/walletSelectorContext';
-import { TextInput, TextInputWithUnits } from '../editor/elements';
+import { useWalletSelector } from "../../contexts/walletSelectorContext";
+import { TextInput, TextInputWithUnits } from "../editor/elements";
 import debounce from "lodash.debounce";
-import './export.scss';
+import "./export.scss";
 
 export default class Export extends Component {
     static contextType = useWalletSelector();
 
     errors = {
-        user: new ArgsError(errorMsg.ERR_INVALID_ADDR, value => ArgsAccount.isValid(value), true),
-        dao: new ArgsError("Invalid address", value => ArgsAccount.isValid(value), true),
-        multicall: new ArgsError("Invalid address", value => ArgsAccount.isValid(value), true),
-        gas: new ArgsError(errorMsg.ERR_INVALID_GAS_AMOUNT, value => ArgsBig.isValid(value) && value.value !== ""),
-        depo: new ArgsError(errorMsg.ERR_INVALID_DEPO_AMOUNT, value => ArgsBig.isValid(value) && value.value !== ""),
-        amount: new ArgsError("Invalid amount", value => ArgsBig.isValid(value) && value.value !== ""),
-        token: new ArgsError("Invalid address", value => ArgsAccount.isValid(value)),
-        desc: new ArgsError("Invalid proposal description", value => value.value !== "", true),
-        noToken: new ArgsError("Address does not belong to token contract", value => this.errors.noToken),
-        notWhitelisted: new ArgsError("Token not whitelisted on multicall instance", value => this.errors.notWhitelisted),
-        hasErrors: new ArgsError("Please fix all errors", value => this.errors.hasErrors)
+        user: new ArgsError(errorMsg.ERR_INVALID_ADDR, (value) => ArgsAccount.isValid(value), true),
+        dao: new ArgsError("Invalid address", (value) => ArgsAccount.isValid(value), true),
+        multicall: new ArgsError("Invalid address", (value) => ArgsAccount.isValid(value), true),
+        gas: new ArgsError(errorMsg.ERR_INVALID_GAS_AMOUNT, (value) => ArgsBig.isValid(value) && value.value !== ""),
+        depo: new ArgsError(errorMsg.ERR_INVALID_DEPO_AMOUNT, (value) => ArgsBig.isValid(value) && value.value !== ""),
+        amount: new ArgsError("Invalid amount", (value) => ArgsBig.isValid(value) && value.value !== ""),
+        token: new ArgsError("Invalid address", (value) => ArgsAccount.isValid(value)),
+        desc: new ArgsError("Invalid proposal description", (value) => value.value !== "", true),
+        noToken: new ArgsError("Address does not belong to token contract", (value) => this.errors.noToken),
+        notWhitelisted: new ArgsError(
+            "Token not whitelisted on multicall instance",
+            (value) => this.errors.notWhitelisted
+        ),
+        hasErrors: new ArgsError("Please fix all errors", (value) => this.errors.hasErrors),
     };
 
     total = {
-        gas: new ArgsBig("267.5", toGas("1"), toGas("270"), "Tgas"),
+        gas: new ArgsBig("270", toGas("1"), toGas("270"), "Tgas"),
```

Comment:
Why did it revert the changes from 87368ecd9f6583af091af0001147f6f9590844a4 ?

Reply:
Oops, seems like merging glitch. I'll fix this

Reply:
Fixed